core/iwasm/libraries/libc-builtin/libc_builtin_wrapper.c

```diff
@@ -1060,6 +1087,10 @@ static NativeSymbol native_symbols_libc_builtin[] = {
     REG_NATIVE_FUNC(malloc, "(i)i"),
     REG_NATIVE_FUNC(realloc, "(ii)i"),
     REG_NATIVE_FUNC(calloc, "(ii)i"),
+#if WASM_ENABLE_SHARED_HEAP != 0
+    REG_NATIVE_FUNC(shared_malloc, "(i)i"),
+    REG_NATIVE_FUNC(shared_free, "(*)"),
+#endif
```

Comment:
i guess this should be a separate lib, not a part of libc_builtin.

Reply:
Got